lib/quic_trace.proto

```diff
@@ -153,6 +153,15 @@ message Event {
   optional TransportState transport_state = 7;
   // For event_type = EXTERNAL_PARAMETERS, record parameters specified.
   optional ExternalNetworkParameters external_network_parameters = 8;
+
+  // Indicates whether the packet sent is a tail loss probe, cf.
+  // https://tools.ietf.org/html/draft-ietf-quic-recovery-14#section-4.3.2
+  optional bool is_tail_loss_probe = 9 [default = false];
```

Comment:
Given there's at least TLP, RTO, and PROBING retransmissions, how about an enum?

Reply:
I'm actually conflicted about this.  I can see RTO and PROBING, but the thing with TLP is that TLP is not even necessarily retransmission...